stats/lib/handler.js

```diff
@@ -92,6 +93,8 @@ const handler = async (req, res, pgPools) => {
     await respond(fetchParticipantChangeRates)
   } else if (req.method === 'GET' && segs[0] === 'participant' && segs[1] && segs[2] === 'scheduled-rewards') {
     await respond(fetchParticipantScheduledRewards, segs[1])
+  } else if (req.method === 'GET' && segs[0] === 'participant' && segs[1] && segs[2] === 'reward-transfers') {
```

Comment:
I think this data is generic (shared by all impact evaluators), it isn't specific to Spark. Therefore, the implementation should go to `stats/lib/platform-routes.js` and `stats/lib/platform-stats-fetcher.js`.

Reply:
https://github.com/filecoin-station/spark-stats/pull/178